Test/git-issues/git-issue-968.dfy.expect

```diff
@@ -1,2 +1,2 @@
-git-issue-968.dfy(11,14): Error: type of type parameter could not be determined; please specify the type explicitly
+git-issue-968.dfy(11,11): Error: type of type parameter could not be determined; please specify the type explicitly
```

Comment:
Here as well I prefer the position of the error to be on the "." where the type parameter should be inserted.

Reply:
It was after the `.` not before the `.`, where the type parameter is expected.